portfolio/src/main/webapp/index.html

```diff
@@ -149,8 +150,24 @@ <h2>Slide Show</h2>
 </div>
 <div id="connect">
   <h2>Connect with me</h2>
-  <p>
-    Follow me here
+  <p>Follow me here
+  <div class="container">
+    <div class="row mt-1">
+      <div class="col">
+        <a href="https://www.linkedin.com/in/prashantneupane" class="btn btn-dark" role="button"
+          rel="noopener noreferrer">LinkedIn</a>
+      </div>
+      <div class="col">
+        <a href="https://www.instagram.com/shady_prashant/" class="btn btn-dark" role="button"
+          rel="noopener noreferrer">Instagram</a>
+      </div>
+      <div class="col">
+        <a href="https://github.com/Prashantneu8848" class="btn btn-dark" role="button"
+          rel="noopener noreferrer">GitHub</a>
+      </div>
+    </div>
+  </div>
```

Comment:
Use `<table>` instead of `<div>`. If you use the same css classes it will look the same and you can omit the ending tags of table elements: `<tr>`, `<td>`, `<th>`, etc. as per https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags

```html
  <table class="container">
    <tr class="row mt-1">
      <td class="col">
        <a href="https://www.linkedin.com/in/prashantneupane" class="btn btn-dark" role="button"
            rel="noopener noreferrer">LinkedIn</a>
      <td class="col">
        <a href="https://www.instagram.com/shady_prashant/" class="btn btn-dark" role="button"
            rel="noopener noreferrer">Instagram</a>
      <td class="col">
        <a href="https://github.com/Prashantneu8848" class="btn btn-dark" role="button"
            rel="noopener noreferrer">GitHub</a>
  </table>
```

This is also easier to read since the element tag more closely reflects its structural purpose.

Reply:
Thanks for the edit.
Changes made.